### ISSUE TITLE
Drawer - update aria-expanded

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-react#4053]
As part of https://github.com/patternfly/patternfly/pull/2897/files, we removed `aria-expanded="true/false"` from the drawer panel element. `aria-expanded` should be used on the element that toggles the drawer panel, not the drawer itself.

* remove `aria-expanded` from the `pf-c-drawer__panel` element
* add `aria-expanded="true/false"` to the buttons that toggle the panel in the component examples

@tlabaj not sure if this is a breaking change? I'm assuming it isn't.
